index.bs

```diff
@@ -49,198 +49,207 @@ conversion domain.
-        `"conversion-data"` field of |url|'s [=url/query=].
-    : [=conversion/conversion time=]
+    : [=attribution trigger/trigger data=]
+    :: The result of applying [=parse attribution data=] with the value associated with the
+        `"data"` field of |url|'s [=url/query=] and the user agent's [=max trigger data value=].
+    : [=attribution trigger/trigger time=]
     :: The current time.
+    : [=attribution trigger/reporting endpoint=]
+    :: |url|'s [=url/origin=]
+
+<dfn>Max trigger data value</dfn> is a vendor specific integer which controls the values of [=attribution-report/trigger-data].
 
 Issue: Formalize how to parse the query similar to URLSearchParams.
 
-<h3 algorithm id="register-conversion">Register a conversion</h3>
+<h3 algorithm id="triggering-attribution">Triggering attribution</h3>
 
-To <dfn>register a conversion</dfn> from a [=request=] |request|, run the following steps:
+To <dfn>trigger attribution</dfn> from a [=request=] |request|, run the following steps:
 
-1. If |request|'s [=request/current url's=] [=url/path=] is not `.well-known/register-conversion`,
+1. If |request|'s [=request/current url's=] [=url/path=] is not `.well-known/attribution-reporting/trigger-attribution`,
     return.
 1. If |request|'s [=request/redirect count=] is less than 1, return.
 1. Let |previousUrl| be the second to last [=URL=] in |request|'s
     [=request/URL list=].
 1. If |request|'s [=request/current url's=] [=url/origin=] is not [=same origin=] with
     |previousUrl|'s [=url/origin=], return.
-1. Let |conversionToRegister| be the result of applying [=create a conversion=] from the
-    request's [=request/current url=].
+1. Let |trigger| be the result of running [=obtain an attribution trigger=] with
+    |request|'s [=request/current url=].
 
     Note: the restriction to require a redirect is needed to ensure that the
-    request's origin is aware and in control of the conversion registration.
+    request's origin is aware and in control of triggering attribution.
 
-1. Issue: Need to spec how to store the conversion.
+1. Issue: Need to spec how to store |trigger|.
 
-<h3 algorithm id="parsing-data">Parsing data fields</h3>
+<h3 algorithm id="parsing-data-fields">Parsing data fields</h3>
 
 This section defines how to parse and extract both
-[=impression/impression data=] and [=conversion/conversion data=] from a
-[=string=] |input| and a unsigned long long |maxData|.
+[=attribution source/event id=] and [=attribution trigger/trigger data=].
 
-<dfn>Parsing conversion data</dfn> from |input| with |maxData| returns the result of
-the following steps:
+To <dfn>parse attribution data</dfn> given a [=string=] |input|, and an unsigned long long
+|maxData| perform the  following steps. They return an unsigned long long:
 
-1. Let |decodedInput| be the result of decoding |input| as a base-16 integer.
+1. Let |decodedInput| be the result of decoding |input| as a base-10 integer.
```

Comment:
This needs to define what happens if the value isn't a base-10 integer. Probably call https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#rules-for-parsing-non-negative-integers and handle the resulting errors.